modules/data.land/R/helpers.R

```diff
@@ -0,0 +1,7 @@
+stop <- function(...) {
```

Comment:
I see your point, though this seems clunky and not [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). Might be a good argument for moving the logger functions into a separate logging package. I don't want to be overly pedantic, just throwing out the idea.